concepts/basics/introduction.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+
+To set off the C++ journey we are starting with variables, function calls, and comments.
+
+## Comments
+
+Comments come in two flavors: single- and multi-line.
+Everything that comes after `//` in a line is ignored by the compiler.
+Multi-line comments are also known as C-style comments. 
+They are surrounded by `/*` and `*/`.
```

Comment:
~~~suggestion
They are surrounded by `/*` and `*/`.
Anything that comes between these will be ignored as well.

```cpp
code; // these words will be ignored
these_words_will_not_be_ignored; /*
but these
   will be */
```
~~~

Or something like that.

Reply:
I'm not sure I can whip up a good example, with the comments being the first item that is explained.